app/components/slack-window.js

```diff
@@ -41,6 +42,11 @@
 			var win = this.getWindow();
 			var bodyEl = win.document.body;
 			var that = this;
+
+			win.addEventListener('contextmenu', function handleContextMenu(evt) {
```

Comment:
Please add a trailing space to the function name to keep consistency with the rest of the code:

```
function handleContextMenu (evt)
```

We do this to denote the name isn't required because the function is inline